test/webhook_test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+sed -ri "s|'(quay.io/reactiveops/polaris:).+'|'\1${CIRCLE_SHA1}'|" ./deploy/webhook.yaml
+kubectl apply &> /dev/null -f ./deploy/webhook.yaml
+sleep 15
+
+kubectl apply &> /dev/null -f test/correctconfig.yaml
```

Comment:
Nit: Although it technically doesn't make a difference, I almost never see this structure in the wild. Normally redirecting output is put at the end of the command for readability. 

`kubectl apply -f test/correctconfig.yaml &>/dev/null` is normally how you would see it in the wild.